sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/implementation/models/AzureSQLConnectionStringParam.java

```diff
@@ -13,7 +13,7 @@
     /*
      * The connection string to access the Azure SQL.
      */
-    @JsonProperty(value = "connectionString", required = true)
+    @JsonProperty(value = "connectionString")
```

Comment:
Is this a swagger update?
Does this change the way SDK's require params when creating a DataFeed?

Reply:
Yes, it's a swagger update. The reason for the service team changing it is, it's an output model as well but in response `connectionString` will be absent. But of course, when creating it is required.